railties/lib/rails/generators/test_unit/scaffold/scaffold_generator.rb

```diff
@@ -17,6 +17,10 @@ def create_test_files
         template_file = options.api? ? "api_functional_test.rb" : "functional_test.rb"
         template template_file,
                  File.join("test/controllers", controller_class_path, "#{controller_file_name}_controller_test.rb")
+
+        unless options.api?
+          template "system_test.rb", File.join("test/system", class_path, "#{file_name.pluralize}_test.rb")
+        end
```

Comment:
Looks like this method is defined in [`Rails::Generators::AppBase`](https://github.com/rails/rails/blob/master/railties/lib/rails/generators/app_base.rb#L404-L406). Would it be best to replicate this in `TestUnit::Generators::Base` or rely on the `system_tests` option set [here](https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/scaffold/scaffold_generator.rb#L19)?